data/bedrock/latest/proto.yml

```diff
@@ -1780,7 +1780,8 @@ packet_command_request:
    # Internal specifies if the command request internal. Setting it to false seems to work and the usage of
    # this field is not known.
    internal: bool
-   # Version is the version of the command that is being executed. This field currently has no purpose or functionality.
+   # Version is the version of the command that is being executed. This field currently has no known purpose
```

Comment:
The comment here is a bit contradictory. The version is important and should be set to a specific value for the version

Reply:
Something like "Specifies the version of the command to run, relative to the current Minecraft version. Should be set to 52 as of 1.19.62" I think would be clearer

Reply:
Sounds good to me. Hopefully this doesn't change per version though.